htpasswd/basic.py

```diff
@@ -8,7 +8,6 @@
 except ImportError:
     from ordereddict import OrderedDict
 
-
```

Comment:
This blank line was here for a reason, don't remove it.

Reply:
Sure, I wasn't aware of it. sorry.  :)